app/kumactl/pkg/util/util.go

```diff
@@ -0,0 +1,24 @@
+package util
+
+import (
+	"fmt"
+	"time"
+)
+
+// Duration returns a readable representation of the provided time
+func Duration(d time.Duration) string {
+	if seconds := int(d.Seconds()); seconds < -1 {
+		return fmt.Sprintf("nil")
+	} else if seconds < 0 {
+		return fmt.Sprintf("0s")
+	} else if seconds < 60 {
+		return fmt.Sprintf("%ds", seconds)
+	} else if minutes := int(d.Minutes()); minutes < 60 {
+		return fmt.Sprintf("%dm", minutes)
+	} else if hours := int(d.Hours()); hours < 24 {
+		return fmt.Sprintf("%dh", hours)
+	} else if hours < 24*365 {
+		return fmt.Sprintf("%dd", hours/24)
+	}
+	return fmt.Sprintf("%dy", int(d.Hours()/24/365))
```

Comment:
Can we inline this function into `TimeSince`?
I don't like putting function as globals (for me utils = globals) just because it _may_ be reused.

Also `util/util.go` does not say much about the code.

Reply:
wrote in a way for reusing it in the future if we want to change inspect data plane's `last connected` in the above format. or can I move the function to a suitable file `duration.go` under util ?

Reply:
Actually it makes sense to use it now. Can we just put this into `Ago()` function and remove `TimeSince()`?

Reply:
Just now checked `inspectDataplanes` `Ago` implementation. it is having `reference` as an argument. can we conclude with a format and proceed?

Reply:
> Just now checked `inspectDataplanes` `Ago` implementation. it is having `reference` as an argument. can we conclude with a format and proceed?

Reply:
yeah, current impl is fine 👍 just some minors about tests.